app/assets/stylesheets/publisher.scss

```diff
@@ -243,14 +237,6 @@
     .hide-location { display: inline-block; }
     .locator { display: none; }
   }
-
-  .twitter-typeahead {
-    left: -1px;
-    // Override inline rule of Typeahead
-    // scss-lint:disable ImportantRule
-    position: absolute !important;
```

Comment:
On the second though, I think that it is more clear to fix the width issue instead of removing `position: absolute`. I'll try to do that now.